.github/workflows/pulsar-ci.yaml

```diff
@@ -104,12 +104,12 @@ jobs:
           distribution: 'temurin'
           java-version: 17
 
-      - name: Check source code license headers
-        run: mvn -B -T 8 -ntp initialize apache-rat:check license:check
+      - name: Check source code license headers, style and spotbugs
+        run: mvn -B -T 8 -ntp initialize apache-rat:check license:check checkstyle:check spotbugs:check
 
       - name: Build core-modules
         run: |
-          mvn -B -T 1C -ntp -Pcore-modules,-main clean install -DskipTests -Dlicense.skip=true -Drat.skip=true
+          mvn -B -T 1C -ntp -Pcore-modules,-main clean install -DskipTests -Dlicense.skip=true -Drat.skip=true -Dcheckstyle.skip -Dspotbugs.skip
```

Comment:
the syntax for skipping doesn't look right. I'd assume `-Dspotbugs.skip=true`. Please validate locally that the commands do what is expected.

Reply:
I usually use that syntax locally